src/views/Main/Schemas/Components/NewEdmObjectInput.js

```diff
@@ -222,13 +224,38 @@ export class NewEdmObjectInput extends React.Component {
             {this.renderInputField('Namespace', NAMESPACE_FIELD)}
             {this.renderInputField('Name', NAME_FIELD)}
             {this.renderInputField('Description', DESCRIPTION_FIELD)}
+            {this.renderPiiDropdown()}
           </div>
         );
       default:
         return null;
     }
   }
 
+  renderPiiDropdown = () => {
+    const options = [
+      { value: true, label: 'True' },
+      { value: false, label: 'False' }
+    ];
+    return (
+      <div>
+        <div>PII</div>
+        <div className={styles.spacerMini} />
+        <Select
+            value={this.state.pii}
+            onChange={this.handlePiiChange}
+            options={options}
+            className={styles.piiSelect} />
+        <div className={styles.spacerSmall} />
+      </div>
+    );
+  }
+
+  handlePiiChange = (e) => {
+    const newValue = (e && e !== undefined) ? e.value : false;
```

Comment:
`(e && e !== undefined)` seems redundant. what are you trying to check for specifically?